ansys-templates edits to doc (#39)
pyansys/ansys-templates: README.rst

> Welcome to PyAnsys Templates

Reviewer: ```suggestion
Welcome to Ansys Templates
```

> Welcome to Ansys Templates
> ============================

Reviewer: ```suggestion
Welcome to Ansys Templates
==========================
```

> The ``pyansys-templates`` repository holds a collection of useful templates compliant

Reviewer: ```suggestion
The ``ansys-templates`` repository holds a collection of useful templates compliant
```

with PyAnsys guidelines. It also provides the ``ansys-templates`` command line tool
for interactively generating new projects based on previous templates.

The main advantages of using this tool are:

- Building process is fully interactive. There is no need to manually modify files.
- Output of the project can easily be customized during the rendering process.
- Generated projects are compliant with `PyAnsys Developer's Guidelines`_.

.. _PyAnsys Developer's Guidelines: https://dev.docs.pyansys.com/

> For information on using this tool, see the `PyAnsys Template Documentation`_.

Reviewer: ```suggestion
For information on using this tool, see the `Ansys Templates Documentation`_.
```

> .. _PyAnsys Template Documentation: https://templates.pyansys.com/ 

Reviewer: ```suggestion
.. _Ansys Templates Documentation: https://templates.pyansys.com/ 
```
